tidb-lightning/tidb-lightning-physical-import-mode-usage.md

```diff
@@ -221,3 +221,26 @@ If the table is large, Lightning will split the table into multiple batches of 1
 
 The [`num-threads`](/tikv-configuration-file.md#num-threads) configuration of TiKV can also affect the performance. For new clusters, it is recommended to set `num-threads` to the number of CPU cores.
+
+## Configure disk quota <span class="version-mark">New in v6.2.0</span>
+
+When you imports data in physical import mode, TiDB Lightning creates a large number of temporary files on the local disk. These files are used to encode, sort, and split the original data. When the local disk space is insufficient, TiDB Lightning reports an error and exits because of write failure.
+
+To avoid this situation, you can configure disk quota for TiDB Lightning. When the temporary files hit disk quota, TiDB Lightning pauses reading the source data and writing temporary files. TiDB Lightning prioritizes writing the sorted key-value pairs to TiKV. After deleting the local temporary files, TiDB Lightning continues the import process.
+
+To enable disk quota, add the following configuration to your configuration file:
+
+```toml
+[tikv-importer]
+# MaxInt64 by default, which is 9223372036854775807 bytes.
+disk-quota = "10GB"
+backend = "local"
+
+[cron]
+# The interval of checking disk quota. 60 seconds by default.
+check-disk-quota = "30s"
+```
+
+`disk-quota` sets the quota for TiDB Lightning. The default value is MaxInt64, which is 9223372036854775807 bytes. This value is much larger than the disk space you might need for the import, so leaving it as the default value is equivalent to not setting disk quota.
+
+`check-disk-quota` is the interval of checking disk quota. The default value is 60 seconds. When TiDB Lightning checks the disk quota, it adds a lock to the relevant data, which pauses all the import threads. Therefore, if TiDB Lightning checks the disk quota before every write, it will significantly slow down the write efficiency (as slow as a single-thread write). To achieve efficient write, disk quota is not checked before every write; every `check-disk-quota` seconds, TiDB Lightning pauses all the import threads and checks the disk quota. That is, if the value of `check-disk-quota` is set to a large value, the disk space used by TiDB Lightning might exceed the disk quota you set, which leaves the disk quota ineffective. Therefore, it is recommended to set the value of `check-disk-quota` to a small value. The specific value range is determined by the environment in which TiDB Lightning is running. In different environments, TiDB Lightning writes temporary files at different speeds. Theoretically, the faster the speed, the smaller the value of `check-disk-quota` should be.
```

Comment:
```suggestion
`check-disk-quota` is the interval of checking disk quota. The default value is 60 seconds. When TiDB Lightning checks the disk quota, it will acquire an exclusive lock for the relevant data, which will block all the import threads. Therefore, if TiDB Lightning checks the disk quota before every write, it will significantly slow down the write efficiency (as slow as a single-thread write). To achieve efficient write, disk quota is not checked before every write; instead, TiDB Lightning pauses all the import threads and checks the disk quota every `check-disk-quota` seconds. That is, if the value of `check-disk-quota` is set to a large value, the disk space used by TiDB Lightning might exceed the disk quota you set, which leaves the disk quota ineffective. Therefore, it is recommended to set the value of `check-disk-quota` to a small value. The specific value of this item is determined by the environment in which TiDB Lightning is running. In different environments, TiDB Lightning writes temporary files at different speeds. Theoretically, the faster the speed, the smaller the value of `check-disk-quota` should be.
```

Reply:
We usually limit the use of "will" in technical documentation. See [Google Documentation Style Guide > Present Tense](https://developers.google.com/style/tense) for reasoning.
The rest of this suggestion looks good. Thanks!

Reply:
Updated in [e5746e8](https://github.com/pingcap/docs/pull/9756/commits/e5746e8f325947379ed4618ed2468e33f57bff35)